### ISSUE TITLE
Input Format for DRMM

### ISSUE THREAD

**Issue by voladorlu** [NTMC-Community/MatchZoo#740]
Hi, can anybody help me on the input format of DRMM? Based on the given tutorial, the cross entropy loss function needs sample negative instances. What if the input data already includes the negative samples (i.e. labeled as 0, positive one labeled as 1)? The negative sampling method will do negative sampling for every instance in the training data (including the negative one labeled as 0)?

**Comment by Crysitna** [comment]
if i get your question right: 
1. the negative sampling can only happen if the negative samples are included in the input otherwise there is nowhere to sample from (see [here](https://github.com/NTMC-Community/MatchZoo/blob/master/matchzoo/data_generator/data_generator.py#L290))
2. yes, it would do negative sampling for each positive data for _num_dup_ times (see [here](https://github.com/NTMC-Community/MatchZoo/blob/master/matchzoo/data_generator/data_generator.py#L280-L291))

**Comment by voladorlu** [comment]
@Crysitna  Thank you so much for your clarification. It really helps me a lot. -:)

**Comment by voladorlu** [comment]
One more question. I note that the default batch_size for DRMM is somehow small (suggested as 20). Will a relative larger batch size (e.g. 128, 256) have negative influence on the model performance?

**Comment by uduse** [comment]
I hope things are working well for you now. I’ll go ahead and close this issue, but I’m happy to continue further discussion whenever needed.
